### ISSUE TITLE
Update Your First Wagtail Site for Wagtail 2.0

### ISSUE THREAD

**Issue by tmfwhovamber** [wagtail/wagtail#4783]
### Issue Summary

Doing a fresh install of Wagtail 2 is impossible because it appears that `start` hasn't been updated from what was in Wagtail 1

Following the directions from **Your First Wagtail Site** https://docs.wagtail.io/en/v2.2.2/getting_started/tutorial.html:

`pip install wagtail` - installs Wagtail 2.2.2
`wagtail start mysite`
`cd mysite`
At this point, my `pip list` looks like this:
![screen shot 2018-09-20 at 12 47 40 pm](https://user-images.githubusercontent.com/11076085/45834173-82e5ff80-bcd4-11e8-9dd3-11f3ab6060b1.png)

`pip install -r requirements.txt`
now a `pip list` looks like this:
![screen shot 2018-09-20 at 12 56 40 pm](https://user-images.githubusercontent.com/11076085/45834277-b7f25200-bcd4-11e8-8393-8fa604c5fe53.png)

Which seems wrong, but it's because Wagtail 2 requirements.txt has:
Django>=1.11,<1.12
wagtail>=1.12,<1.13
in it, which seems kinda sucky as I want to work in Wagtail 2.2.2 and the default settings are set to a previous version and not even Wagtail 2

Then I also see:
![screen shot 2018-09-20 at 12 53 31 pm](https://user-images.githubusercontent.com/11076085/45834316-cb9db880-bcd4-11e8-803f-25455bbb0215.png)

Which seems real wrong, because I know all those name spaces changed

### Steps to Reproduce

1. fresh install of `pip install wagtail` in a completely new python 3 environment
2. `wagtail start my_new_wagtail_2_project`
3. take a look at your new project's requirements file
![screen shot 2018-09-20 at 12 36 06 pm](https://user-images.githubusercontent.com/11076085/45833099-cbe88480-bcd1-11e8-917c-3c464471e102.png)
4. pip list
5. pip install -r requirements.txt 
6. pip list again

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: Yes

### Technical details
Initially:
* Python version: 3.6.4
* Django version: 2.0.8
* Wagtail version: 2.2.2
* Browser version: it doesn't matter because I'm not even running this yet.

**Comment by tomdyson** [comment]
Somehow you have an old `requirements.txt`. The Wagtail 2.2.2 version looks like this:

```
Django>=2.0,<2.1
wagtail>=2.2,<2.3
```

See https://github.com/wagtail/wagtail/blob/stable/2.2.x/wagtail/project_template/requirements.txt

Did you run the all the commands in the same virtualenv? How are you creating your virtualenv?

**Comment by tmfwhovamber** [comment]
I have created completely new virtualenvs each time with virtualenvwrapper https://virtualenvwrapper.readthedocs.io/en/latest/install.html

I started on this quest because I was trying to duplicate a problem with the text editor not showing up in my project where I've upgraded to python 3, django 2, and Wagtail 2. So, I switched from a Wagtail 2 (python 3, django 2) editor to a Wagtail 2 editor. I haven't been anywhere near Wagtail 1 code for about two/three weeks. I don't know where I would have picked it up.

**Comment by tmfwhovamber** [comment]
Ok, I was TOTALLY thinking that I just did this wrong somehow - maybe I'd created a virtualenv without passing which python and it defaulted to python 2, and somehow that defaulted something to Wagtail 1.# and resulted in some weird requirements thing. So, I destroyed all of those virtualenvs, I destroyed all of the new wagtail site directories, and started completely fresh with a virtualenv with Python 3
![screen shot 2018-09-20 at 1 56 25 pm](https://user-images.githubusercontent.com/11076085/45837545-96e22f00-bcdd-11e8-80e0-c0f041128b4a.png)
with nothing in it:
![screen shot 2018-09-20 at 1 56 33 pm](https://user-images.githubusercontent.com/11076085/45837581-a792a500-bcdd-11e8-9977-fd3ea7986d21.png)
and then `pip install wagtail` results in:
![screen shot 2018-09-20 at 1 57 06 pm](https://user-images.githubusercontent.com/11076085/45837618-b416fd80-bcdd-11e8-85c7-71ae906e04cc.png)
Then I `vim -R requirements.txt`:
![screen shot 2018-09-20 at 1 58 00 pm](https://user-images.githubusercontent.com/11076085/45837642-c2651980-bcdd-11e8-8688-d347689160dd.png)
What the heck?!?!?!

**Comment by tmfwhovamber** [comment]
Additionally, I've got this in settings.base.py:
![screen shot 2018-09-20 at 2 28 16 pm](https://user-images.githubusercontent.com/11076085/45839133-7c11b980-bce1-11e8-86b7-8ee30377dc28.png)

**Comment by tomdyson** [comment]
@tmfwhovamber could you please go through these steps, and report back with the output from the `pip list`, `which wagtail` and `less requirements.txt` commands?

```
cd /tmp
mkvirtualenv --python=python3 dispose-wagtail222
pip list
pip install wagtail
pip list
which wagtail
wagtail start foo
cd foo
less requirements.txt
``` 

to clear up afterwards, you can do

```
deactive
rmvirtualenv dispose-wagtail222
rm -rf /tmp/foo
```

**Comment by tmfwhovamber** [comment]
![screen shot 2018-09-21 at 10 00 28 am](https://user-images.githubusercontent.com/11076085/45885827-7968a000-bd85-11e8-91a6-831c9d6fa07e.png)
![screen shot 2018-09-21 at 10 00 44 am](https://user-images.githubusercontent.com/11076085/45885829-7c639080-bd85-11e8-8d31-c5a63f50ed17.png)
![screen shot 2018-09-21 at 10 00 55 am](https://user-images.githubusercontent.com/11076085/45885837-7e2d5400-bd85-11e8-83db-f0ab2a3bf107.png)
![screen shot 2018-09-21 at 10 01 14 am](https://user-images.githubusercontent.com/11076085/45885845-808fae00-bd85-11e8-9642-4c9cfc52325d.png)
![screen shot 2018-09-21 at 10 01 21 am](https://user-images.githubusercontent.com/11076085/45885850-82597180-bd85-11e8-9580-a692d43f8975.png)
![screen shot 2018-09-21 at 10 00 11 am](https://user-images.githubusercontent.com/11076085/45885851-84bbcb80-bd85-11e8-8064-295b669686a4.png)

**Comment by tmfwhovamber** [comment]
Ok, so I figured this out! Apparently the requirements were being pulled from my globally installed Wagtail 1.12.2 and not my locally installed virtualenv Wagtail 2.0. So, when I uninstalled wagtail globally,  rm the Wagtail app made with it, and made a new one with the locally installed Wagtail, then it pulled the requirements folder appropriately. 
I guess, lesson is: make sure that you don't have Wagtail globally installed. 
Thanks for all your help, @tomdyson!
